builtInFunctions/esdtNFTCreate.go

```diff
@@ -432,6 +461,29 @@ func getNonceKey(tokenID []byte) []byte {
 	return append(noncePrefix, tokenID...)
 }
 
+func (e *esdtNFTCreate) validateEsdtType(esdtType uint32) error {
+	if _, isValid := e.validEsdtTypes[esdtType]; !isValid {
+		return fmt.Errorf("%w, invalid esdt type", ErrInvalidArguments)
```

Comment:
Can we also add in this error the wrong `esdtType` ?

Reply:
Added